.env.example

```diff
@@ -1,4 +1,11 @@
-FLEEK_API_KEY=[FLEEK API KEY GOES HERE]
-FLEEK_API_SECRET=[FLEEK API SECRET GOES HERE]
-INFURA_URL=[INFURA URL GOES HERE]
-ZAUCTION_ADDRESS=[ZAUCTION ADDRESS GOES HERE]
\ No newline at end of file
+FLEEK_STORAGE_API_SECRET=
```

Comment:
You removed all of the `storage/` code that uses this, so might as well remove all the related ENV vars too